exonum-light-client/README.md

```diff
@@ -1,8 +1,30 @@
 # Exonum Java Light Client
 Java client for [Exonum blockchain][exonum].
 
-## Quickstart
-If you are using Maven, add this to your pom.xml file
+## Overview
+Exonum light client is Java library for working with Exonum blockchain
+from the client side and can be easily integrated to existing 
+Java application.  
+Also, Exonum light client provides an access to [common utils][ejb-common]
+toolkit which contains some helpful functions for _hashing_,
+_serialization_ ect. 
+
+## Prerequisites
+- Java 8 or above is required for using this client
+- Maven 3.5 or above (only if you need to build it locally)
+
+## How to build
```

Comment:
Let's push it to the bottom — you don't need it unless you are willing to contribute.